### PR TITLE
The VS and dot is seprated

### PR DESCRIPTION
**Pull request by yu8833**
Signed-off-by: YuPengZTE yu.peng36@zte.com.cn

### CODE REVIEW

File: docs/userguide/storagedriver/overlayfs-driver.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Since version 1.12, Docker also provides `overlay2` storage driver which is much
 more efficient than `overlay` in terms of inode utilization. The `overlay2`
 driver is only compatible with Linux kernel 4.0 and later.
 
-For comparison between `overlay` vs `overlay2`, please also refer to [Select a
+For comparison between `overlay` vs. `overlay2`, please also refer to [Select a
```
Comment (thaJeztah) [inline docs/userguide/storagedriver/overlayfs-driver.md]: I think `vs` should actually be "and" here ("comparison between Foo and Bar")

**Review by thaJeztah — CHANGES_REQUESTED** [review]
one suggestion, but looks good once addressed

File: docs/userguide/storagedriver/overlayfs-driver.md (hunk 1 above)
Comment (YuPengZTE) [inline docs/userguide/storagedriver/overlayfs-driver.md]: Thanks @thaJeztah 
The 'between and' is better, and I see 'please also refer to [Select a
storage driver](selectadriver.md#overlay-vs-overlay2)', so is it better that 'For comparison between `overlay` vs `overlay2`' is changed to 'For  `overlay` vs `overlay2`' ?

Comment (thaJeztah) [inline docs/userguide/storagedriver/overlayfs-driver.md]: I think I actually prefer 

```Markdown
For a comparison between `overlay` and `overlay2`,
please refer to ....
```

Just `For overlay vs overlay2` is not very descriptive

Comment (justincormack) [inline docs/userguide/storagedriver/overlayfs-driver.md]: The `.` is definitely incorrect here anyway. You don't put a dot unless there are omitted letters at the end. And between .. versus is wrong too. Use `and`.

Comment (thaJeztah) [inline docs/userguide/storagedriver/overlayfs-driver.md]: Seems that US English uses `vs.`, and British English uses `vs` (looked that up earlier)

Comment (YuPengZTE) [inline docs/userguide/storagedriver/overlayfs-driver.md]: Thanks @thaJeztah and @justincormack .
PTAL.

**Comment by mstanleyjones** [comment]
The best fix is to write around 'vs'. It is Latin, and it's not great to include Latin in docs that will be consumed by people with different levels of comfort in English. Why not change these headings to "Differences between A and B"? If you must use "versus", just spell it out. However, I suspect that these changes may conflict with PR #23791

**Comment by thaJeztah** [comment]
I'm not sure it's worth changing this (just adding the `.`), also the docs/userguide/storagedriver/overlayfs-driver.md file is now published from the https://github.com/docker/docker.github.io repository, so I think it's best to close this PR.

Thanks for contributing though @YuPengZTE !

**Pull request closed without merging**